pkgs/development/compilers/go/1.7.nix

```diff
@@ -116,7 +116,7 @@ stdenv.mkDerivation rec {
       })
     ];
 
-  SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
+  NIX_SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
```

Comment:
Yeah, it's unclear to me why Go would need a certificate bundle at build time. Is it going to download something via https?